File: calculator.py
from select import select

def add(num1,num2):
    return num1 + num2

def subtract(num1,num2):
    return num1 - num2

def multiply(num1,num2):
    return num1 * num2

def divide(num1, num2):
    return num1 / num2 

print("Please choose a valid operation")
print("Enter + for addition:")
print("Enter - for subtraction:")
print("Enter * for multiplication:")
print("Enter / for division:")

select = input("Select an operator from +, -, *, / :")
print("Enter two numbers")
number_1 = int(input())
number_2 = int(input())

if select == "+":
    print("The result is: ",add(number_1,number_2))

elif select == "-":
    print("The result is: ", subtract(number_1,number_2))

elif select == "*":
    print("The result is: ", multiply(number_1,number_2))

elif select == "/":
    print(divide("The result is: ",number_1,number_2))

else:
    print("Invalid input")

File: calculator.java
import java.util.Scanner;
public class calculator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        char operator;
        boolean userChoice = true;
        Double num1, num2, result;
        System.out.println("Please choose a valid operation");
        System.out.println(" Enter + for addition: ");
        System.out.println(" Enter - for subtraction: ");
        System.out.println(" Enter * for multiplication: ");
        System.out.println(" Enter / for division: ");
        operator = input.next().charAt(0);
        System.out.println("Enter two numbers");
        num1 = input.nextDouble();
        num2 = input.nextDouble();
        switch(operator) {
            case '+':
            result = num1 + num2;                
            System.out.println("The result is: " + result);
            break;

            case '-':
            result = num1 - num2;                
            System.out.println("The result is: " + result);
            break;

            case '*': 
            result = num1 * num2;
            System.out.println("The result is: " + result);                
            break;

            case '/':
            result = num1 / num2;
            System.out.println("The result is: " + result);
            break;

            default:
            System.out.println("Invalid operation");
        }
        input.close(); 
    }
}